setup.py

```diff
@@ -7,7 +7,6 @@
     install_requires=[
         'psycopg2-binary',
         'sqlalchemy',
-        'py2neo',
         'progressbar',
```

Comment:
Can we drop some of these other requirements? I think you deleted the only thing that was using progressbar, and I don't see anything using avro.